README.md

```diff
@@ -130,6 +130,10 @@ See [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTE
 #### [Testing]
 
 To test the software, you can use the standard `pytest` tool from the root of the directory.
+
+To install the dependencies for testing, you can run `pip install .[google,testing,sensors,focuser]` from the root
```

Comment:
Many shells will have problems with the `[` without quoting so let's just put this in there.

```suggestion
To install the dependencies for testing, you can run `pip install ".[google,testing,sensors,focuser]"` from the root
```